src/sqlfluff/rules/L068.py

```diff
@@ -0,0 +1,258 @@
+"""Implementation of Rule L068."""
+
+from typing import List, Optional
+
+from sqlfluff.core.parser import WhitespaceSegment
+
+from sqlfluff.core.parser import BaseSegment
+from sqlfluff.core.rules.base import BaseRule, LintFix, LintResult, RuleContext
+from sqlfluff.core.rules.doc_decorators import (
+    document_configuration,
+    document_fix_compatible,
+    document_groups,
+)
+import sqlfluff.core.rules.functional.segment_predicates as sp
+
+
+@document_groups
+@document_configuration
+@document_fix_compatible
+class Rule_L068(BaseRule):
+    """The brackets should stand at the same line with from/as/join keywords.
```

Comment:
Have you tested how this rule interacts with other rules such as L003 (indentation) and L016 (line length), perhaps others? I think there's potential for unexpected rule interactions.